wp-content/themes/pub/wporg-learn-2024/src/course-outline/index.js

```diff
@@ -4,10 +4,39 @@ import { Icon, drafts, lockOutline } from '@wordpress/icons';
 import { renderToString } from '@wordpress/element';
 
 document.addEventListener( 'DOMContentLoaded', () => {
+	/**
+	 * Allow the entire header to toggle, giving users a larger area to interact with.
+	 */
+	document
+		.querySelectorAll( 'section.wp-block-sensei-lms-course-outline-module-bordered > header' )
+		.forEach( ( header ) => {
+			const button = header.querySelector( 'button' );
+			header.addEventListener( 'click', () => {
+				button.click();
+			} );
+
+			button.addEventListener( 'click', ( event ) => {
+				event.stopPropagation();
+			} );
+
+			// To enable the entire header to be clickable for toggling without conflicts, remove the link.
+			// In fact, this link duplicates the first lesson link in the course outline.
+			// See https://github.com/WordPress/Learn/pull/2776#issuecomment-2258308422
+			const link = header.querySelector( 'h2 > a' );
+			if ( link ) {
+				const heading = link.parentElement;
+				heading.innerHTML = link.innerHTML;
```

Comment:
The title used in the screen reader text is also a link, can you replace that as well?

Reply:
Which title you're referring to? I don't seem able to locate it..

![image](https://github.com/user-attachments/assets/cb1f6df6-faca-4bd4-985e-2fccb9fff7c7)

Reply:
Inside the button, it's invisible text for screen readers.

<img width="723" alt="Screenshot 2024-07-30 at 8 20 50 PM" src="https://github.com/user-attachments/assets/91c8f7de-e7cb-461a-af50-b2653cf5a222">

Reply:
ah.. didn't notice that! 54b5fd0

Reply:
| Before | After |
|-|-|
| ![image](https://github.com/user-attachments/assets/dfab2135-6bba-431f-8a96-f13de414962c) | ![image](https://github.com/user-attachments/assets/c0843ac9-3cbb-4f1d-b891-3b701383e4d2) |